src/styles/searchResults.css

```diff
@@ -0,0 +1,30 @@
+.masonry-grid {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 30px;
+  width: 100%;
+  max-width: 1200px;
+}
+
+.masonry-grid_column {
+  flex: 1;
+  min-width: 250px;
+  background-clip: padding-box;
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+}
+
+.image {
```

Comment:
Would be a good idea to rename this class so its clear what component its being used in, `search-results__image` for example